server.py

```
@@ -86,48 +99,61 @@ def index():
         .order_by(Articles.created_at.desc())
         .all()
     )
+
+    products = (
+        (Products.query.filter(Products.in_stock != False))
```

Ruff (0.12.2)

Avoid inequality comparisons to `False`; use `Products.in_stock:` for truth checks

Replace with `Products.in_stock`

(E712)